src/features/TakeNow/ui/ValidateTakeNowParams.tsx

```diff
@@ -31,13 +31,13 @@ function ValidateTakeNowParams({
       const multiInformantState = getMultiInformantState();
       if (
         !isInMultiInformantFlow() ||
-        sourceSubject.id !== multiInformantState.sourceSubject?.id ||
-        targetSubject.id !== multiInformantState.targetSubject?.id
+        sourceSubject.id !== multiInformantState?.sourceSubject?.id ||
+        targetSubject.id !== multiInformantState?.targetSubject?.id
       ) {
         initiateTakeNow({ sourceSubject, targetSubject });
       }
     }
-  }, [data, initiateTakeNow, isInMultiInformantFlow, isSuccess]);
+  }, [data, getMultiInformantState, initiateTakeNow, isInMultiInformantFlow, isSuccess]);
```

Comment:
I believe we should leave the dependency arrays as-is. Since these changes are unrelated to the bug fix, they can be added to dev instead

Reply:
Thank you, makes sense. Done